priv/prelude/mnesial.specs.erl

```diff
@@ -0,0 +1,164 @@
+-module(mnesia).
```

Comment:
Why is the filename mnesial.specs.erl? Should it not be mnesia.specs.erl?

Reply:
Oops, you're right of course! (The file name is not checked/used so it makes no functional difference but still.)